sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,141 @@
+
+# Specification
+
+The _current_ STX emission schedule is presented as follows.  Note that the **first STX halving is in December 2024**.  The tail emission after the final halving in 2050 would be 125 STX per block, and the total supply at that time is projected to be 1,783,063,600 STX.
+
+| Coinbase Reward Reduction Phase | Approximate Date (time b/w halvings) | STX Reward (reduction) | STX Supply (after) |
+|--------------------------------|-------------------------------------|----------------------|-------------------|
+| Current                         | -                                   | 1000                 | -                  |
+| 1st                             | ~Dec 2024 (4 yrs)                  | 500 (50%)           | 1,512,993,315      |
+| 2nd                             | ~Dec 2028 (4 yrs)                  | 250 (50%)           | 1,645,084,888      |
+| 3rd                             | ~Dec 2032 (4 yrs)                  | 125 (50%)           | 1,689,389,675      |
+| -                               | ~Jan 2050 (17.08 yrs)             | 125 (0%)             | 1,783,063,600      |
+
+
+The _proposed_ STX emission schedule is presented as follows.  In particular, this SIP proposes preserving the 1000 STX coinbase until April 2026.  After this, there would be a reduction to 500 STX after two years, and another reduction to 125 STX after two more years.  The tail emission after the final halving in 2050 would be 100 STX, and the total supply at that time is projected to be 1,783,063,600 STX (about 0.19% lower).
+
+
+| Coinbase Reward Reduction Phase | Bitcoin Block Height | Approximate Date (time b/w halvings) | STX Reward (reduction) | STX Supply (after) |
+|--------------------------------|---------------------|-------------------------------------|----------------------|-------------------|
+| Current                        | -                   | -                                   | 1000                 | -                 |
+| 1st*                     | 945,000             | ~April 2026 (+1.33 yrs)            | 500 (50%)           | 1,607,907,038     |
+| 2nd                     | 1,050,000           | ~April 2028 (2 yrs)                | 250 (50%)           | 1,652,242,188     |
+| 3rd                      | 1,260,000           | ~April 2032 (4 yrs)                | 125 (50%)           | 1,696,547,013     |
+| 4th                      | 1,470,000           | ~April 2036 (4 yrs)                | 100 (20%)           | 1,718,699,425     |
+| -                          | -                   | ~Jan 2050 (13.833 yrs)             | 100 (0%)             | 1,779,628,415     |
+
+
+With these changes, miner incentives and PoX yield remain unchanged for another 16 months, which we believe is sufficient time for the nascent Nakamoto signer set to develop into a set of stable, professional block signers, and for the sBTC project to attract sufficient initial liquidity.
+
```

Comment:
Missing 'Related Work' section: The SIP must include a 'Related Work' section that summarizes alternative solutions addressing similar problems and explains why they are not adequate.

Reply:
@will-corcoran you can add links to the reports here, that should be sufficient, or explain why this section isn't relevant for this SIP

Reply:
since i'm not aware of any precedence here @will-corcoran  - you may just want to include that we're not aware of any related work in this area. i've looked and cannot find anything

Reply:
@wileyj @jiga related work added

Reply:
thank you @will-corcoran - very helpful to see the proposed STX emission schedule